src/components/shared/Subscribe.vue

```diff
@@ -86,8 +90,8 @@ export default {
     </div>
     <span>olarak almak için</span>
     <input type="text" v-model="name" placeholder="İsim soyisim" class="username">
-    <input type="text" v-model="email" placeholder="Email">
-    <button class="button" @click="handleSubscribe">
+    <input @keyup.enter.once="handleEnter" type="text" v-model="email" placeholder="Email">
```

Comment:
You can also replace this `handleEnter` with `handleSubscribe ` since it doesn't do anything rather than calling `handleSubscribe`. We don't need `handleEnter` at all.

Reply:
Removed handleEnter.